rest_framework/utils/encoders.py

```diff
@@ -65,3 +68,57 @@ def default(self, obj):
         elif hasattr(obj, '__iter__'):
             return tuple(item for item in obj)
         return super().default(obj)
+
+
+class NestedMultiPartEncoder:
```

Comment:
considering the fact that django support writable nested serialization, this type of functionality should be the default to align. but I want to know what are the counter logics